### ISSUE TITLE
Don't allow the use of "language" tags on literals of the observation table.

### ISSUE THREAD

**Issue by l00mi** [zazuko/cube-creator#635]
Textual strings should always where possible added as concepts. For test cases, and as preparation for a shared dimension it is allowed to keep the string, it should not be allowed to have a language.

A 'concept' table on the other hand should allow to use language tags for the mapping.

**Comment by AFoletti** [comment]
I am not sure I understand this one correctly.
You mean that in the observation table I am not allowed to use literals, but only concepts that link to another table containing the literal (possibly multilanguage) description of the concept itself?

**Comment by l00mi** [comment]
Yes, that is the idea. It is documented, and this issue would add a dynamic warning. It would still be possible for tests and similar.

**Comment by AFoletti** [comment]
Thanks for the clarification.
I am actually unsure about that. While a good idea conceptually, it may be too restrictive. What if I want to add (possibly multilanguage) annotations to my observations? I would need a second table with an entry for each observation, which does not look that elegant to me.

**Comment by l00mi** [comment]
Do you know that you can create concep tables directly from the sam csv as the cube (observations) table? 

In the output data the concepts with multi lingual labels are necessary for filter management purposes.

**Comment by AFoletti** [comment]
> Do you know that you can create concep tables directly from the sam csv as the cube (observations) table?

You mean in the cube creator UI? No, I was not aware of that and I am really interested in how this works.

**Comment by l00mi** [comment]
You can simply select (e.g. only the remark fields) from the CSV and create a new table.

**Comment by AFoletti** [comment]
> You can simply select (e.g. only the remark fields) from the CSV and create a new table. I will keep an eye to document this better.
> 
thanks!

> Also, in the specific case of remarks,annotations per observation it might make sense to have it directly attached to the observation, we will discuss this when we introduce annotations more specificaly.

Makes sense to me.
